infra/cifuzz/cifuzz.py

```diff
@@ -524,6 +524,13 @@ def is_project_sanitizer(sanitizer, oss_fuzz_project_name):
 
   # Simple parse to prevent adding pyYAML dependency.
   with open(project_yaml, 'r') as file_handle:
-    if sanitizer + '\n' in file_handle.read():
-      return True
+    file_data = file_handle.read()
+
+  # TODO(metzman): Replace this with proper handling of project.yaml files.
+  if 'sanitizers:\n' not in file_data:
+    logging.info('No sanitizers defined in project.yaml, assuming %s is valid.',
+                 sanitizer)
+    return True
```

Comment:
I wonder if it would make sense to replace it with `return sanitizer in ['address', 'undefined']`?

Reply:
Sure. Done.

Reply:
I'd probably add a comment with a link to https://google.github.io/oss-fuzz/getting-started/new-project-guide/#sanitizers somewhere to make it easier to figure out where the list comes from. Had I known that the list of sanitizers is optional, I would probably have fixed it in https://github.com/google/oss-fuzz/pull/3969 without breaking the h2o project.